dla_cnn/data_model/Sightline.py

```diff
@@ -3,24 +3,26 @@
 
 class Sightline(object):
 
-    def __init__(self, id, dlas=None, flux=None, loglam=None, z_qso=None):
+    def __init__(self, id, ra=None,dec=None,dlas=None, flux=None, wavelength=None,error=None, z_qso=None):
         """
 
         Args:
             id (int):  Index identifier for the sightline
             dlas (list): List of DLAs
             flux (np.ndarray):
-            loglam (np.ndarray):
-                log10 of observed wavelength values
+            wavelength (np.ndarray):
```

Comment:
Revert back to `loglam` instead of `wavelength`

Reply:
I have changed it back.